app/views/optOut/ConfirmedOptOut.scala.html

```diff
@@ -31,8 +31,7 @@
     govukInsetText : GovukInsetText
 )
 
-@(viewModel: ConfirmedOptOutViewModel, isAgent: Boolean = false)(implicit messages: Messages, user: auth.MtdItUser[_])
-
+@(viewModel: ConfirmedOptOutViewModel, isAgent: Boolean = false, showReportingFrequencyContent: Boolean)(implicit messages: Messages, user: auth.MtdItUser[_])
```

Comment:
we already have isAgent details in user/(in MtdItUser type), so this is a duplicate flag, can be removed;
+ we might need to raise a tech debt to discuss if we need these details across rest of the views

Reply:
Hi Pavel, as it wasn't something that was introduced in my work, I feel it's out of scope of the ticket (as it'll require a bit of refactoring and as you mentioned there are more occurrences of this too). So I'll get a ticket made up for tech debt 😃